bibliopixel/drivers/driver_base.py

```diff
@@ -107,6 +134,11 @@ def _send_packet(self):
         pass
 
     def update_colors(self):
+        """Apply any corrections to the current color list
+        and send the results to the driver output. This function primarily
```

Comment:
We generally use the word "push" to send things to drivers.  On the other hand, we never actually sat down and discussed the terminology, so don't worry too much.